codersdk/audit.go

```diff
@@ -72,6 +72,7 @@ const (
 	AuditActionLogin    AuditAction = "login"
 	AuditActionLogout   AuditAction = "logout"
 	AuditActionRegister AuditAction = "register"
+	AuditActionDormant  AuditAction = "dormant"
```

Comment:
I feel like if you have a `dormant` action then you need something for becoming active again.

Reply:
Honestly i forgot I added this, I just implemented it with ActionWrite